app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field regist-field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] @Team Great work adding the registration page, however the design slightly differs from the design given , check the comparison below:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/katarighe/blog-app/assets/61408860/e99fbec7-d31c-41b2-bd32-e6e01f9284bd) | ![image](https://github.com/rbd3/Blog-App2/assets/61408860/cb818577-82e9-4610-a5de-ed2c69a9a38b)

Reply:
- [ ] @Team I was unable to sign up as a new user even after filling are the fields in the form, am getting the error below, Kindly consider checking your devise setup to ensure the login and registration work correctly.
![image](https://github.com/rbd3/Blog-App2/assets/61408860/236ea538-53fa-403a-a1d9-c7454a7edf80)